### DISCUSSION TITLE
Make XPath on document addition more helpful

### DISCUSSION THREAD

**Discussion by madoleary** [tosdr/edit.tosdr.org#1023] [Ideas]
We should explain why we're asking for xpath, and we should also give a couple of examples with their own succinct explanations of format.

**Comment by arthurlutz** [comment]
Would recommending and extension to help find the Xpath such as https://addons.mozilla.org/en-US/firefox/addon/xpath_finder/ be helpful ? 

So far, am a bit disappointed with the said extension since it find xpath with indices for elements which might break when the page changes.
